plugins/inline.py

```diff
@@ -32,12 +32,13 @@ async def answer(bot, query):
                            switch_pm_parameter="hehe")
         return
 
-    if AUTH_CHANNEL and not await is_subscribed(bot, query):
+    invite_links = await is_subscribed(bot, query=query)
+    if AUTH_CHANNEL and len(invite_links) >= 1:
         await query.answer(results=[],
-                           cache_time=0,
-                           switch_pm_text='You have to subscribe my channel to use the bot',
-                           switch_pm_parameter="subscribe")
-        return
+            cache_time=0,
+            switch_pm_text='You have to subscribe my channel to use the bot',
+            switch_pm_parameter="subscribe")
+    return
```

Comment:
Just add the new block spaces here, I by mistake remove and it may coz return without running full code 🙃.